win-odbc64/odbcclient/Drvr35Res/Drvr35Res_os.vcxproj

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Line 122 in this file should be deleted since Drvr35Res.rc2 is removed?
     <None Include="res\Drvr35Res.rc2" />